fr/orm/saving-data.rst

```diff
@@ -36,6 +36,10 @@ créer une nouvelle entity et de la passer à la méthode ``save()`` de la class
     // L'$article et l'entity $savedArticle retourné contient maintenant l'id
     $id = $savedArticle->id;
 
+En utilisant ``newEntity``, la méthode ``isNew()`` de l'entity nouvellement 
+crée renverra ``true``. C'est ainsi que, lors de l'appel de ``save()``, 
+CakePHP saura qu'il s'agit d'un ajout et non une mise à jour.
+
```

Comment:
All this paragraph doesn't exis in the original en version and is already explain twice in this page.